Packs/FortiGate/TestPlaybooks/playbook-Fortigate_Test.yml

```diff
@@ -1,1213 +1,5674 @@
-id: Fortigate Test
+id: FortiGate
```

Comment:
Why did you change the id?

Reply:
I've copied all of the other files content and replaced the current one as you asked.
I guess change was also included in it.
Would you like me to change the `name` and `id`?

Reply:
changed `id` and `name` back to the previous value.

Reply:
Look at the conf.json file it already contains the Fortigate_Test PB, also your yml file 
![image](https://github.com/demisto/content/assets/95565843/f74f38c4-c8f1-4809-99f0-44425f6aadf6)
Keep the original id/name/display